### ISSUE TITLE
Consider labeling streets by default

### ISSUE THREAD

**Issue by kostajh** [savaslabs/durham-restaurants-map#6]
I like the toggle. I prefer to have the streets labeled by default, especially since the map is somewhat abstract, so it's easier to see where things are.

**Comment by timstallmann** [comment]
Open to this, but I think it looks prettier w/o them initially -- @AnneTee @dmurphy1 @chrisarusso what do y'all think?

**Comment by AnneTee** [comment]
I agree that it's prettier without them initially, and it's possible that people will be more drawn to a beautiful map than an immediately functional one. Interested to hear other's thoughts.

I'll admit that I didn't even notice the "label streets" button at first - can I suggest a bit of a border radius (2-3 px) on the buttons to make them look more like buttons?

**Comment by timstallmann** [comment]
Nice call @AnneTee a 2px border radius looks good!
